aliquotmaf/merging/record_merger/impl/v1_0.py

```diff
@@ -185,7 +185,22 @@ def maf_from_first_element(
                 maf_dic[column] = get_builder(column, self.scheme, value=vals)
 
             else:
-                maf_dic[column] = selected_caller[column]
+                # NOTE: Not a complete solution
+                try:
+                    maf_dic[column] = selected_caller[column]
+                except KeyError as e:
+                    if e.args == ('RNA_Support',):
```

Comment:
This is awkward because it's just a lazy hack to support those new columns which we don't even have an annotator for, can discuss other ways for sure.